### PR TITLE
Documentation of findcoords

### PR DESCRIPTION
**Pull request by Daniel-Tojal**
Documentation of the function findcoords. Description, INPUTS and OUTPUT in need of completion

### CODE REVIEW

File: ea_findcoords.m [hunk 1]
```diff
@@ -1,5 +1,25 @@
 function coords=ea_findcoords(goodz,trajectory,trajvector,dist,correction,options)
-
+%
```
Comment (Daniel-Tojal) [inline ea_findcoords.m]: Please provide a description of what this function is doing

File: ea_findcoords.m [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+%
+% USAGE:
+%
+%    coords = ea_findcoords(goodz,trajectory,trajvector,dist,correction,options)
+%
+% INPUTS:
```
Comment (Daniel-Tojal) [inline ea_findcoords.m]: Please provide a description of the inputs

File: ea_findcoords.m [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+%    goodz:
+%    trajectory:
+%    trejvector:
+%    dist:
+%    correction:
+%    options:
+%
+% OUTPUT:
```
Comment (Daniel-Tojal) [inline ea_findcoords.m]: Please provide a description of the outputs.

File: ea_findcoords.m (hunk 1 above)
Comment (ningfei) [reply ea_findcoords.m]: Seems deprecated, need input from @andreashorn

**Comment by laurentheirendt** [comment]
@andreashorn, we need your input here ...

**Pull request closed without merging**